src/ostree/ot-remote-builtin-delete-cookie.c

```diff
@@ -69,6 +70,52 @@ ot_remote_builtin_delete_cookie (int argc, char **argv, GCancellable *cancellabl
   cookie_file = g_strdup_printf ("%s.cookies.txt", remote_name);
   jar_path = g_build_filename (gs_file_get_path_cached (repo->repodir), cookie_file, NULL);
 
+#ifdef HAVE_LIBCURL
```

Comment:
Can we put these implementations in their own functions in `ot-remote-cookie-util.c`?